builtins/amp-img.js

```diff
@@ -16,6 +16,8 @@
 
 import {BaseElement} from '../src/base-element';
 import {dev} from '../src/log';
+import {isExperimentOn} from '../src/experiments';
+import {guaranteeSrcForSrcsetUnsupportedBrowsers} from '../src/utils/img';
```

Comment:
This and other calls to guaranteeSrcForSrcsetUnsupportedBrowsers were not implemented by me. It's just a few lines, but idk how important it is to ensure the changes that show up are only yours, so just wanted to point it out in case there is something I should do to fix it!